src/bindings/channel_header.ts

```diff
@@ -0,0 +1,34 @@
+import {AppBinding} from 'mattermost-redux/types/apps';
+
+import {ZDIcon, Routes, newChannelHeaderBindings, Bindings} from '../utils';
+
+// getChannelHeaderBindings returns the users slash command bindings
```

Comment:
why "slash command"?

Reply:
I just missed changing the comment while probably copying from the slash_command.ts file.  Fixed.